src/contracts/GPv2Settlement.sol

```diff
@@ -133,6 +133,21 @@ contract GPv2Settlement {
     }
 
     // solhint-enable
+    modifier onlyOwner {
```

Comment:
will we have a different notion of owner and solver?

Reply:
Yes surely, In fact we may not even wind up using the term owner. We were thinking of going with something like "Authorized Parties"